tests/tests_suite_test.go

```diff
@@ -364,6 +366,18 @@ func waitUntilDeployed() {
 	deploymentTimedOut = false
 }
 
+func waitUntilTemplateValidatorIsRunning() {
+	key := client.ObjectKey{Name: validator.VirtTemplateValidator, Namespace: strategy.GetNamespace()}
```

Comment:
I think it would be clearer if the `Name` is `validator.DeploymentName`. Then, if we choose to change the name in the future, the test would be correct.